easybuild/easyconfigs/c/CFITSIO/CFITSIO-3.48-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'ConfigureMake'
+
+name = 'CFITSIO'
+version = '3.48'
+
+homepage = 'https://heasarc.gsfc.nasa.gov/fitsio/'
+description = """CFITSIO is a library of C and Fortran subroutines for reading and writing data files in
+FITS (Flexible Image Transport System) data format."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/']
+sources = ['%%(namelower)s-%s.tar.gz' % version]
```

Comment:
Thank you for moving `sources` below `source_urls`. This is how it should always be. We should have a test blocking PRs that do not follow this rule :slightly_smiling_face: